src/Operators/finitedifference.jl

```diff
@@ -4032,7 +4032,9 @@ function _serial_copyto!(field_out::Field, bc, Ni::Int, Nj::Int, Nh::Int)
     space = axes(field_out)
     bounds = window_bounds(space, bc)
     bcs = bc # strip_space(bc, space)
+    mask = Spaces.get_mask(axes(field_out))
     @inbounds for h in 1:Nh, j in 1:Nj, i in 1:Ni
+        DataLayouts.compute(mask, CartesianIndex(i, j, 1, 1, h)) || continue
```

Comment:
Just curious, why do you use `continue` here instead of `return nothing`?

Reply:
Because we're in a loop, we want to continue to the next iteration of the loop.

Reply:
For cuda kernels, there is no more work to be done by the thread, so we can just `return nothing`.